cmd/crl-checker/main.go

```diff
@@ -73,8 +75,17 @@ func main() {
 	err = json.Unmarshal(urlFileContents, &urls)
 	cmd.FailOnError(err, "Parsing JSON Array of CRL URLs")
 
-	issuer, err := issuance.LoadCertificate(*issuerFile)
-	cmd.FailOnError(err, "Loading issuer certificate")
+	if *issuerFile == "" {
+		cmd.Fail("-issuer is required, but may be '-' to disable validation")
```

Comment:
Modify the description of this flag to reflect that it is required but can be disabled with `'-'`

Reply:
Done!